### PR TITLE
Fix error when downloading whisper model

### PR DESCRIPTION
**Pull request by KIRA009**
**What kind of change does this PR introduce?**
This PR addresses #815 

**Summary**
This PR fixes the exception that was being raised when downloading the whisper model for the first time in an application. Whisper internally uses `tqdm` for the download progress bar, which was causing issues when there was no `sys.stdout` present.

**Checklist**
* [x] My code follows the style guidelines of OpenAdapt
* [x] I have performed a self-review of my code
* [ ] If applicable, I have added tests to prove my fix is functional/effective
* [x] I have linted my code locally prior to submission
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have made corresponding changes to the documentation (e.g. README.md, requirements.txt)
* [x] New and existing unit tests pass locally with my changes

**How can your code be run and tested?**
To test this, first reproduce the bug
- Checkout to `main` branch
- Go to the path output by `os.path.join(os.path.expanduser("~"), ".cache")` and delete the `whisper` directory if its present.
- Build the app using `python -m openadapt.build` and then run the built app and start then stop recording. You should see the error window.

Then checkout to this branch, delete the `whisper` directory again, then build and run the app, start and stop the recording. The recording should stop without errors.

### CODE REVIEW

File: openadapt/app/dashboard/app/recordings/page.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ export default function Recordings() {
     }, []);
     return (
         <Box>
-            {/* {recordingStatus === RecordingStatus.RECORDING && (
+            {recordingStatus === RecordingStatus.RECORDING && (
```
Comment (abrichr) [inline openadapt/app/dashboard/app/recordings/page.tsx:53]: @KIRA009 is this intentional? It seems unrelated to the rest of the PR.

**Review by abrichr — COMMENTED** [review]
Thank you @KIRA009 !

File: openadapt/app/dashboard/app/recordings/page.tsx (hunk 1 above)
Comment (KIRA009) [reply openadapt/app/dashboard/app/recordings/page.tsx:53]: @abrichr Yes, since the reason for commenting this out has been fixed, I have added it back

**Review by abrichr — APPROVED** [review]
LGTM!
